packages/ingest/package.json

```diff
@@ -26,10 +26,10 @@
     "CUMULUS",
     "NASA"
   ],
-  "homepage": "https://github.com/cumulus-nasa/cumulus/tree/master/packages/ingest",
+  "homepage": "https://github.com/cumulus/cumulus/tree/master/packages/ingest",
```

Comment:
Looks like you nuked the wrong part: I think it should be `https://github.com/nasa/cumulus/tree/master/packages/ingest`